config/database.yml

```diff
@@ -83,5 +81,3 @@ test:
 #
 production:
   <<: *default
-  url: <%= ENV['HEROKU_POSTGRESQL_CHARCOAL_URL'] %>
-  password: <%= ENV['CSQUIZ_DATABASE_PASSWORD'] %>
```

Comment:
Heroku injects db config automatically when building rails apps, so should be no need for these lines.

Note the current lead database url with credentials is also aliased as "DATABASE_URL" to facilitate promoting another db.